lab.R

```diff
@@ -63,19 +63,19 @@ pop = read_csv(file.path('data', 'county_population.csv'))
 data("nytcovcounty")
 #' *Use tools such as `skimr::skim()` to answer the following questions.  Remember that, if you use any packages in the final script, you should (a) use a `library()` call at the top to load them and (b) add them to the list in the `DESCRIPTION` file (remember the comma between items).*
 #' 
-
+skimr::skim(nytcovcounty)
```

Comment:
As the instructions indicate, this needs to be added to DESCRIPTION or Travis won't know to install it and will raise an error